{{cookiecutter.project_slug}}/Makefile

```diff
@@ -26,23 +27,23 @@ style: flake8 isort license
 
 ## Run the tests.
 test:
-	docker-compose run --rm -e ENVIRONMENT=testing web \
+	-docker-compose run --rm -e ENVIRONMENT=testing web \
```

Comment:
Not sure what the dash is for.

Reply:
It tells Make to keep going even if the command returns with a non-zero exit code. The use case is to be able to run all QA targets locally, and not stop on the first error.